specification/keyvault/Security.KeyVault.Keys/client.tsp

```diff
@@ -17,6 +17,15 @@ using KeyVault;
 @@clientName(JsonWebKeyType, "KeyType", "java");
 
 // Go configuration
+@@clientName(getDeletedKeys, "ListDeletedKeyProperties", "go");
+@@clientName(getKeys, "ListKeyProperties", "go");
+@@clientName(getKeyVersions, "ListKeyPropertiesVersions", "go");
+// renaming Keytem to KeyProperties.
+// renaming existing KeyProperties to something else
+// to avoid a naming conflict as it's not part of the public surface area
+@@clientName(KeyProperties, "ShouldNotGenerate", "go");
```

Comment:
How does this work?

Reply:
Not adding anything new here,  just moving things around. Have to rename KeyProperties to something else because I'm renaming KeyItem to KeyProperties. If I don't, there will be a name conflict.